frontend/src/mocks/msw/handlers/announcementLists/announcementListHandlers.js

```diff
@@ -0,0 +1,16 @@
+import { rest } from 'msw';
+import { announcementListResponse } from './announcementListResponse';
```

Comment:
```suggestion
import { announcementsResponse } from 'mocks/msw/handlers/announcements/announcementResponse';
```

Niżej napisałem dlaczego